frontend/packages/dev-console/src/components/project-access/project-access-form-submit-utils.ts

```diff
@@ -73,3 +76,14 @@ export const sendRoleBindingRequest = (verb: string, roles: UserRoleBinding[], r
   });
   return finalArray;
 };
+
+export const getGroupedRole = (role: UserRoleBinding, roleBindings: RoleBinding[]) => {
+  let groupedRole: RoleBinding;
+  _.forEach(roleBindings, (roleBinding: RoleBinding) => {
+    if (roleBinding.metadata.name === role.roleBindingName && roleBinding.subjects.length > 1) {
+      _.remove(roleBinding.subjects, (subject) => subject.name === role.user);
```

Comment:
Hmm, not a big fan of changing input parameters if a 'get' helper function. Esp in this case where getGroupedRole was called with some props we should avoid this.

@debsmita1 Did you see an option that this method return new values instead of modifying the input variables?

Reply:
fixed it